### ISSUE TITLE
Bottom row shows bright "afterglow" for blue pixels

### ISSUE THREAD

**Issue by danielmader** [benjohnemmett/Hub75MicroPython#3]
Thank you again for your fix with the SPI MOSI pins and the new naming convention! (I've done similar modification to your sources while trying to find the bug, but I'll now stick with your work.)

Now, there's only one thing remaining for me: the bottom line shows a bright "afterglow" on its previous blue pixels.

I'll attach your src with docstrings and a modified main.py with my pinout and a TODO insert to show the problem.

Again, I am clueless here, but I guess the problem is similar.
[src.zip](https://github.com/benjohnemmett/Hub75MicroPython/files/10830970/src.zip)

**Comment by benjohnemmett** [comment]
Resolved on main branch.

**Comment by benjohnemmett** [comment]
The afterglow was due to a separate issue that the last pixel issue. The afterglow was happening because the blue pixel data that was shifted in for the last row was still sitting there between frames. I just added a few lines of code to flush it out at the end of the display_data() method.

Again, thanks for the feedback and help cleaning up the code. I also included the doc strings you added in this update as well.
